Ginger/Ginger/Variables/ValidationRule/DateTimeFormatValidationRule.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Globalization;
+using System.Windows.Controls;
+
+namespace Ginger.Variables
+{
+    public class DateTimeFormatValidationRule : ValidationRule
+    {
+        private DateTime dateTimevalue;
+        public DateTimeFormatValidationRule(DateTime value)
+        {
+            dateTimevalue = value;
+        }
+
+        public override ValidationResult Validate(object value, CultureInfo cultureInfo)
+        {
```

Comment:
Validation needs to cover the format validation as well.

Reply:
first we set the format with intial datetime field and it will gives the string with changed format and then converting the string into datetime, if format is valid it will pass the validation else will throw invalid date format error.